proms-resources/frontend/src/main/frontend/src/index.jsx

```diff
@@ -68,7 +79,19 @@ function PromsHomepage (props) {
   return (<>
     <AppBar position="static" className={classes.appbar}>
       <Toolbar variant="dense" className={classes.toolbar}>
-        <img src="/libs/cards/resources/logo.png" alt="logo" className={classes.logo} />
+        <div>
+          <img src="/libs/cards/resources/logo.png" alt="logo" className={classes.logo} />
+          <Link
+            component="button"
+            color="inherit"
+            variant="body2"
+            underline="always"
+            onClick={() => {setShowTou(true);}}
+            className={classes.tou}
+          >
+            Terms of Use
+          </Link>
+        </div>
```

Comment:
The layout doesn't look great. Let's make "Terms of Use" centered in the header. IIRC the toolbar should already have flex display and removing the wrapping div should be enough for the change below (removing the wrapper div) to work. Also please remove the underline.
```suggestion
        <img src="/libs/cards/resources/logo.png" alt="logo" className={classes.logo} />
        <Link
          component="button"
          color="inherit"
          variant="body2"
          underline="none"
          onClick={() => {setShowTou(true);}}
          className={classes.tou}
        >
          Terms of Use
        </Link>
```

![image](https://user-images.githubusercontent.com/651980/148438348-f97f83dc-712e-4831-abf9-6b528d2e8945.png)